doc/manual/manual.tex

```diff
@@ -5057,6 +5057,52 @@ \subsubsection{Artificial viscosity smoothing}
 
 
 
+\subsubsection{Tracking finite strain}
+\label{sec:finite-strain}
+\textit{This section was contributed by Juliane Dannberg}
+
+In many geophysical settings, material properties, and in particular the rheology, do not only depend
+on the current temperature, pressure and strain rate, but also on the history of the system. 
+This can be incorporated in a geodynamic model by tracking the strain that accumulates over time
+at every point in the model. Tracking this finite strain can either be done by using compositional
+fields (one field for each component of strain) or by using particles. 
+
+Here, we use a material model plugin that defines the compositional fields as the components of the strain 
+tensor $\varepsilon_{ij} = \frac{\partial v_i}{\partial x_j}$, and modifies the right-hand side of 
+the corresponding advection equations to accumulate strain over time. This is done by adjusting the 
+\verb!out.reaction_terms! variable:
+\lstinputlisting[language=c++]{cookbooks/finite_strain/finite_strain.cc}
+
+The right-hand side term consists of two parts:
+The first one, $R \varepsilon R^T$, rotates the accumulated strain from all the previous time steps with
+the rotation part of the flow field of the current time step, so that strain the material undergoes in the
+current time step can be applied to the correct components. The second part $\dot\varepsilon \Delta t$
```

Comment:
In our papers, and in the manual, we don't actually use `\varepsilon` as a variable of its own, but always as an operator that simply takes the symmetric gradient. So maybe doing something like `\varepsilon(\mathbf U)` would work, where we say that `\mathbf U` is the displacement vector so that `\partial\mathbf U/\partial t=\mathbf u`? Or maybe use `\mathbf d` for the displacement? Then you could use `\varepsilon(\mathbf d)`.

In principle, I may agree that using `\varepsilon` and `\dot\varepsilon` would be desirable. But that ship has sailed several years ago.